### PR TITLE
Remove usage of caf::atom_value for logger configuration

### PR DESCRIPTION
**Pull request by 6yozo**
Removed usage of caf::atom_value for logger configuration.   

### :memo: Checklist

- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

File by file review.

### CODE REVIEW

**Review by tobim — APPROVED** [review]
This look good. I left one comment that needs to be addressed. Feel free to merge once it is resolved.

File: libvast/src/logger.cpp [hunk 1]
```diff
@@ -27,6 +26,8 @@
 #include <spdlog/sinks/rotating_file_sink.h>
 #include <spdlog/sinks/syslog_sink.h>
```
Comment (dominiklohmann) [inline libvast/src/logger.cpp:28]: One additional request: Please rebase and adjust the first commit message to use active voice present tense. A good guideline is that `When applied, this commit will <commit-message>` should be a full sentence without a stop at the end.

Additionally, please apply the same rule to the story title.

Comment (6yozo) [reply libvast/src/logger.cpp:28]: The story title in Shortcut is different but follows the rule.

**Review by dominiklohmann — APPROVED** [review]
Ship it! :shipit:

**Pull request merged**